src/strongbox/addon.clj

```diff
@@ -368,7 +401,32 @@
 
 (defn-spec re-installable? boolean?
   "returns `true` if given `addon` can be re-installed to its current `:installed-version`."
-  [addon map?] ;; deliberately lenient. it's called directly from the gui
+  [addon map?] ;; deliberately lenient, called directly from the gui
```

Comment:
being called from the gui is not a good excuse for leniency. it's lenient because the check is robust and will be used against almost anything. remove these

Reply:
tests